server/src/main/java/keywhiz/service/resources/admin/SecretsResource.java

```diff
@@ -211,7 +211,7 @@ protected SanitizedSecret retrieveSecret(@Auth User user, String name) {
   @Consumes(APPLICATION_JSON)
   public Response createSecret(@Auth User user, @Valid CreateSecretRequestV2 request) {
 
-    logger.info("User '{}' creating secret '{}'.", user, request.name());
+    logger.info("User '{}' creating secret '{}'.", user, request);
```

Comment:
Possibly controversial: rather than logging just the secret name (which provides very little useful information) we log the entire request, **however**, note that the secret contents are redacted (and there are existing tests to enforce this). My reasoning was that this would have made it very trivial to determine whether the problem was inside Keywhiz or on the wire (i.e. somewhere in the serialization/deserialization process) or in the caller.

Reply:
As long as we redact the secret contents, I don't see a problem with this.

Reply:
Is there any other metadata in the secret object that would be concerning (or at least make us pause)?